### PR TITLE
Chore/fix worker request error message

### PR DESCRIPTION
**Pull request by melohagan**
## Description
We were doing a `await response.text()` within a try/catch which is not valid: https://github.com/node-fetch/node-fetch/issues/533

Fixed to check for response header first.

### CODE REVIEW

File: packages/server/src/utilities/workerRequests.ts [hunk 1]
```diff
@@ -40,25 +40,21 @@ export function request(ctx?: Ctx, request?: any) {
...
+    } else {
+      responseErrorMessage = await response.text()
     }
-    const msg = `Unable to ${errorMsg} - ${
-      error.message ? error.message : error
-    }`
+    const msg = `Unable to ${errorMsg} - ${responseErrorMessage}`
     if (ctx) {
-      ctx.throw(400, msg)
+      ctx.throw(msg, response.status)
```
Comment (samwho) [inline packages/server/src/utilities/workerRequests.ts:57]:
```suggestion
      ctx.throw(response.status, msg)
```

Comment (melohagan) [reply packages/server/src/utilities/workerRequests.ts:57]: It actually has to be the other way around to make use of the option status and default to 500:

```ts
/**
         * Throw an error with `msg` and optional `status`
         * defaulting to 500. Note that these are user-level
         * errors, and the message may be exposed to the client.
         *
         *    this.throw(403)
         *    this.throw('name required', 400)
         *    this.throw(400, 'name required')
         *    this.throw('something exploded')
         *    this.throw(new Error('invalid'), 400);
         *    this.throw(400, new Error('invalid'));
         *
         * See: https://github.com/jshttp/http-errors
         */
        throw(message: string, code?: number, properties?: {}): never;
```

Comment (samwho) [reply packages/server/src/utilities/workerRequests.ts:57]: That... is absolutely terrifying.

Comment (melohagan) [reply packages/server/src/utilities/workerRequests.ts:57]: I've never seen anything like it to be fair 😂 

Funny code comment as well: https://github.com/jshttp/http-errors/blob/206aa2c15635dc1212c06c279540972aa90e23ea/index.js#L50

**Review by samwho — APPROVED** [review]

**Pull request merged**